spiffworkflow-frontend/src/components/ProcessSearch.tsx

```diff
@@ -17,14 +17,17 @@ export default function ProcessSearch({
   processes,
   selectedItem,
   onChange,
-  titleText = 'Process model',
+  titleText = 'Process Search',
```

Comment:
i think this also gets used where you're selecting the model on the filter page, so Process model makes more sense there, but probably not a dealbreaker.

Reply:
i take it back, i guess this is a new component.